### ISSUE TITLE
NullPointerException running `console` in SBT

### ISSUE THREAD

**Issue by avdv** [scalacenter/scalafix#362]
Hi.

I have 
```
addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.5.0")
```
in my global plugins.sbt and when running the `console` task in a SBT project, there is this NPE:
```
[play-project@2.6.5] (master) $ console
[info] Starting scala interpreter...
[info] 
Welcome to Scala 2.12.3 (OpenJDK 64-Bit Server VM, Java 1.8.0_144).
Type in expressions for evaluation. Or try :help.

scala> [error] (run-main-0) java.lang.NullPointerException

java.lang.NullPointerException
	at scala.meta.internal.SemanticdbPipeline.$anonfun$scalametaTargetroot$1(ScalahostPipeline.scala:19)
	at scala.Option.map(Option.scala:146)
	at scala.meta.internal.SemanticdbPipeline.scalametaTargetroot(ScalahostPipeline.scala:19)
	at scala.meta.internal.SemanticdbPipeline.scalametaTargetroot$(ScalahostPipeline.scala:16)
	at scala.meta.internal.SemanticdbPlugin.scalametaTargetroot$lzycompute(ScalahostPlugin.scala:9)
	at scala.meta.internal.SemanticdbPlugin.scalametaTargetroot(ScalahostPlugin.scala:9)
	at scala.meta.internal.SemanticdbPipeline$PersistSemanticdbComponent$PersistSemanticdbPhase.run(ScalahostPipeline.scala:83)
	at scala.tools.nsc.Global$Run.compileUnitsInternal(Global.scala:1431)
	at scala.tools.nsc.Global$Run.compileUnits(Global.scala:1416)
	at scala.tools.nsc.Global$Run.compileSources(Global.scala:1412)
	at scala.tools.nsc.interpreter.IMain._initialize(IMain.scala:127)
	at scala.tools.nsc.interpreter.IMain.initializeSynchronous(IMain.scala:147)
	at scala.tools.nsc.interpreter.ILoop.$anonfun$process$12(ILoop.scala:1050)
	at scala.tools.nsc.interpreter.ILoop.startup$1(ILoop.scala:1017)
	at scala.tools.nsc.interpreter.ILoop.$anonfun$process$1(ILoop.scala:1067)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:949)
	at scala.tools.nsc.InterpreterLoop.main(InterpreterLoop.scala:16)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:107)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:82)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:748)
[error] (Thread-3007) java.lang.InterruptedException

java.lang.InterruptedException
	at java.util.concurrent.SynchronousQueue.put(SynchronousQueue.java:879)
	at scala.tools.nsc.interpreter.SplashLoop.run(InteractiveReader.scala:81)
	at java.lang.Thread.run(Thread.java:748)
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.Defaults$$anonfun$consoleTask$1$$anonfun$apply$56.apply(Defaults.scala:877)
	at sbt.Defaults$$anonfun$consoleTask$1$$anonfun$apply$56.apply(Defaults.scala:877)
	at scala.Option.foreach(Option.scala:236)
	at sbt.Defaults$$anonfun$consoleTask$1.apply(Defaults.scala:877)
	at sbt.Defaults$$anonfun$consoleTask$1.apply(Defaults.scala:872)
	at scala.Function8$$anonfun$tupled$1.apply(Function8.scala:35)
	at scala.Function8$$anonfun$tupled$1.apply(Function8.scala:34)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[error] (compile:console) Nonzero exit code: 1
[error] Total time: 93 s, completed Sep 19, 2017 9:16:02 AM
[play-project@2.6.5] (master) $ 
```

Anything I can do to help debug this exception?

**Comment by olafurpg** [comment]
Thank you for reporting! This is a known issue https://github.com/scalameta/scalameta/issues/926 with a pending fix in https://github.com/xeno-by/scalameta/commit/fa1563ca76be40409af48056cb8b77c67b8a70fb

**Comment by olafurpg** [comment]
Fixed in #371 by upgrading scalameta
